src/standalone/Vegetation/pfts.jl

```diff
@@ -0,0 +1,491 @@
+"""
+This file contains the PFT struct used to prescribe vegetation parameters to the 
+model when running using PFTs, as well as the default PFT scheme. ClimaLand 
+supports running using default PFTs, user defined PFTs, or fully prescribed 
+vegetation parameters.
+
+The default PFTs used in the model use parameter values drawn from defferent 
+sources in the literature. The sources are listed here with abbreviations that
+are used in the comments in the code below.
+
+    Chen, J. M., G. Mo, J. Pisek, J. Liu, F. Deng, M. Ishizawa, 
+    and D. Chan (2012), Effects of foliage clumping on the estimation of global
+    terrestrial gross primary productivity, Global Biogeochem. Cycles, 26, 
+    GB1019, doi:10.1029/2010GB003996.
+    abb. as Chen et al. 2012
+
+    CLM5.0 Technical Note
+    abb. as CLM5.0
+
+    CLM2 PFT Data
+    https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/pftdata/
+    abb. as CLM2 PFT Data
+"""
+
+export Pft, default_pfts, params_from_pfts
+
+using DataFrames
+
+# List of parameters that PFTs need to define in order to be valid
+param_list = [
+    # TwoStreamModel parameters
+    :Ω,          # Clumping Index
+    :α_PAR_leaf, # PAR Leaf Reflectance
+    :α_NIR_leaf, # Near-infrared Leaf Reflectance
+    :τ_PAR_leaf, # PAR Leaf Transmittance
+    :τ_NIR_leaf, # Near-infrared Leaf Transmittance
+
+    # Conductance Model
+    :g1, # Medlyn CO2 Sensitivity
+
+    # Photosynthesis model
+    :Vcmax25, # Max rate of carboxylation of Rubisco
+
+    # Plant Hydraulics and general plant parameters
+    :SAI,             # Avg Stem Area Index
+    :f_root_to_shoot, # Root to Shoot Ratio
+    :K_sat_plant,     # Hydraulic Conductivity at Saturation
+    :ψ63,             # Xylem water potential at 63% loss of conductivity
+    :capacity,        # Water Storage Capacity
```

Comment:
I think that this is a derived parameter, and the fundamental parameter is the plant porosity,
rather than:
    `plant_ν =
        params.plant_hydraulics.capacity / (maxLAI * setup.h_leaf) / FT(1000)`
we should solve for capacity given nu, max LAI, h_leaf.

Reply:
Got it